core/field_dropdown.js

```diff
@@ -101,7 +102,12 @@ Blockly.FieldDropdown.prototype.init = function() {
  * @private
  */
 Blockly.FieldDropdown.prototype.showEditor_ = function() {
-  Blockly.WidgetDiv.show(this, this.sourceBlock_.RTL, null);
+  // If there is an existing drop-down we own, this is a request to hide the drop-down.
```

Comment:
(updated this code - I think it was a mistake copied from WidgetDiv at some point).